sdk/python/feast/feature_store.py

```diff
@@ -303,12 +311,18 @@ def get_historical_features(
         except FeatureViewNotFoundException as e:
             sys.exit(e)
 
+        _feature_refs: List[str]
+        if isinstance(feature_refs[0], FeatureService):
+            _feature_refs = _get_features_refs_from_feature_services(feature_refs)  # type: ignore
```

Comment:
Is there a reason why types are ignored?

Reply:
There was but now that I'm chaging to passing in a single FeatureService this shouldn't be an issue any more.